src/common/consts/tokens.ts

```diff
@@ -44,6 +44,25 @@ export const WRAPPED_NATIVE_CURRENCIES: Record<SupportedChainId, TokenInfo> = {
     chainId: SupportedChainId.SEPOLIA,
     address: '0xfFf9976782d46CC05630D1f6eBAb18b2324d6B14',
   },
+
+  [SupportedChainId.POLYGON]: {
+    ...wrappedNativeCurrencyEth,
```

Comment:
`...wrappedNativeCurrencyEth` looks redundant here, all the fields beisdes `decimals` are defined bellow